tests/test_backend.py

```diff
@@ -1,18 +1,20 @@
 # test_backend.py
 
 import errno
-import os
+import pathlib
 import platform
 import re
+import shutil
 import subprocess
 
 import pytest
 
 from graphviz.backend import (run, render, pipe, unflatten, version, view,
                               ExecutableNotFound, RequiredArgumentError)
+import utils
 
```

Comment:
let's add a newlie before it

Reply:
I'll do it, although I am used to three blocks separated by empty line (stdlib, third-party, project). But in this case it is a test, so we may consider the project part to have to sub-parts: project and tests.